src/Teams/Controllers/ManageTeamsController.cs

```diff
@@ -21,5 +30,19 @@ public IActionResult Error()
         {
             return View(new ErrorViewModel { RequestId = Activity.Current?.Id ?? HttpContext.TraceIdentifier });
         }
+
+        [Authorize]
+        public async Task<IActionResult> RemoveAsync(int team_id)
```

Comment:
You can skip Async naming convention on endpoint method level as for the methods in controller you should use default naming convention